.github/scripts/commons.mjs

```diff
@@ -8,17 +8,20 @@ export const config = {
   testAssetsDir: `./test-assets/`,
 }
 
+export const branchName = process.env.BRANCH_NAME || 'main';
```

Comment:
The default branch where we make changes and PRs is `develop`, not `main`. Is it normal that the default here is `main`? (I don't know much the regular deploy flow, so maybe it is).

Reply:
the act-rules.github.io main branch is 'develop', but the w3c/wcag-act-rules repo's is 'main'. Since this is to push either to W3C/wcag-act-rules:main or to any PR on w3c/wcag-act-rules, I think 'main' should be the default.